react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+export function AppDrawer() {
+  const [clicked, setClicked] = useState(false);
```

Comment:
`clicked` isn't a very descriptive name. I can click it numerous times and it changes between true and false. What does `clicked` mean to you? Maybe `isOpen` or something like that would be more desriptive.